src/main/charts/bitbucket/values.yaml

```diff
@@ -104,27 +104,32 @@ bitbucket:
 
   resources:
     jvm:
-      # -- The maximum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the ElasticSearch JVM.
+      # -- JVM memory arguments below are based on the defaults defined for the Bitbucket docker container, see:
+      # https://bitbucket.org/atlassian-docker/docker-atlassian-bitbucket-server/src/master/
+      #
+      # -- The maximum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the Elasticsearch JVM.
       maxHeap: "1g"
-      # -- The minimum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the ElasticSearch JVM.
-      minHeap: "1g"
+      # -- The minimum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the Elasticsearch JVM.
+      minHeap: "512m"
 
     # -- Specifies the standard Kubernetes resource requests and/or limits for the Bitbucket container.
     # It is important that if the memory resources are specified here, they must allow for the size of the Bitbucket JVM.
     # That means the maximum heap size, the reserved code cache size, plus other JVM overheads, must be accommodated.
-    # Allowing for (maxHeap+codeCache)*1.5 would be an example.
-    container: {}
+    # Allowing for maxHeap * 1.5 would be an example.
+    container:
     #  limits:
-    #    cpu: "4"
-    #    memory: "2G"
-    #  requests:
-    #    cpu: "4"
+    #    cpu: "1"
     #    memory: "2G"
+      requests:
+        # -- If changing the cpu value update additional JVM arg 'ActiveProcessorCount' below
+        cpu: "1"
```

Comment:
Where is the default 1 CPU coming from? I believe in the docker environment the containers have access to all cores available to the Docker engine - e.g. the number of cores enabled in the Resources tab in the Docker for Mac settings.

I think restricting to exactly 1 CPU is not necessary - can we put 2 at least here?

Reply:
We decided to settle on 1 cpu based on the testing we did with the updated memory requests. 1 cpu actually performed well, granted this is based on basic testing with little load. I have no problem upping to 2

Reply:
I confirm when we set the cpu request to 1 and don't define the cpu limit, pod have access to all core available in node just like Docker. 
So, we may are safe to leave it as it is also I have no objection to increase it to 2 cores. In this case we may good to increase the memory request and max heap size accordingly by another gigabyte to cover a bit more than minimum dataset for products.